### PR TITLE
Bugfix 5710/Fix Group Menu

### PR DESCRIPTION
**Pull request by PhotoNomad0**
#### Changes
- fixes for badge ordering and alignment

#### Testing
- should pass on Travis

This change is [Reviewable](https://reviewable.io/reviews/translationcoreapps/tc-ui-toolkit/189)

### CODE REVIEW

File: src/GroupedMenu/FilteredMenu/index.js [hunk 1]
```diff
@@ -32,22 +31,18 @@ class FilteredMenu extends React.Component {
    * @param {object} filter - the filter being toggled
    */
   handleToggleFilter = filter => {
```
Comment (neutrinog) [inline src/GroupedMenu/FilteredMenu/index.js]: Why don't you order the filters here using `this.props.filters` as a reference?
That way you don't have to add an extra `order` key to the filter objects?

Comment (PhotoNomad0) [reply src/GroupedMenu/FilteredMenu/index.js]: Good idea

File: README.md [hunk 2]
```diff
@@ -35,8 +35,7 @@ class App extends Component {
 
   or
 - `npm i`
-- `npm run build`
```
Comment (mannycolon) [inline README.md]: You still need this step `npm run build`

Comment (PhotoNomad0) [reply README.md]: Thanks for catching that.

**Pull request merged**